### PR TITLE
VDS-200, VDS-79: Fix errors with invalid input qty values

### PR DESCRIPTION
**Pull request by trueboroda**
Also disable add to cart button if qty = 0
image

image

### CODE REVIEW

File: assets/js/directives/only-integer.js [hunk 1]
```diff
@@ -0,0 +1,22 @@
+var storefrontApp = angular.module('storefrontApp');
+
+storefrontApp.directive('onlyInteger', function () {
```
Comment (pushnitsa) [inline assets/js/directives/only-integer.js:3]: I doubt about a directive naming.
Maybe smth like vc-available-integers-only?
@asvishnyakov what do you think?

Comment (ilyawzrd) [reply assets/js/directives/only-integer.js:3]: i think we had the same directive in odt and meds themes with 'onlyInteger' name

Comment (pushnitsa) [reply assets/js/directives/only-integer.js:3]: Well, if it is a really naming issue, I'd prefer to rename it anyway.

Comment (trueboroda) [reply assets/js/directives/only-integer.js:3]: No problem. I`ll rename this.

**Review by asvishnyakov — APPROVED** [review]

**Pull request merged**